policy/lib/tekton/task_results.rego

```diff
@@ -0,0 +1,89 @@
+package lib.tkn
+
+import rego.v1
+
+# handle the output artifacts from Tekton Chains
+# https://tekton.dev/docs/chains/slsa-provenance/#output-artifacts
+
+# returns the value of a task result with name IMAGE_URL
+task_result_url(task) := value if {
```

Comment:
Might be a bit more straightforward if this was named `task_result_artifact_url`?